src/UI/Http/Rest/Controller/User/UserChangeEmailController.php

```diff
@@ -39,4 +44,22 @@ public function __invoke(string $uuid, Request $request): JsonResponse
 
         return JsonResponse::create();
     }
+
+    private function validateUuid(string $uuid): void
```

Comment:
@Lutacon this method is private but the purpose is more important than the constructor, so what's why I use to add it at the top

Reply:
So the logic is not based on the order by the visibility of the method. I ask that question because of PHP-CS-Fixer and the option 'ordered_class_elements' to do it automatically.

Reply:
Meh... Something to disable hehe

Reply:
😢

Reply:
I know it's not an standard. If you've any thoughts share it to discuss about it because it's something I don't mind to change. 😉